ctest.h

```diff
@@ -391,7 +419,7 @@ static void color_print(const char* color, const char* text) {
         printf("%s\n", text);
 }
 
-#ifdef __APPLE__
+#if defined (__APPLE__) && !defined (_POSIX_C_SOURCE)
```

Comment:
So with POSIX compatibility we lose setup and teardown on Mac? If we don't, is this code necessary at all?